src/ORM/Filters/ExactMatchFilter.php

```diff
@@ -205,4 +221,38 @@ public function isEmpty()
     {
         return $this->getValue() === [] || $this->getValue() === null || $this->getValue() === '';
     }
+
+    /**
+     * Determine if we can use placeholders for the given column and values
+     * Current rules are to use placeholders for all values, unless:
+     * - The column is a DBPrimaryKey or DBForeignKey
+     * - The values being filtered are all either integers or valid integer strings
+     * - Using placeholders for integer ids has been configured off, which is the default config value
+     *
+     * Disabling placeholders large `WHERE ID in (id1, id2, ...) style queries was measured to be significantly
+     * faster if placeholders were disabled when querying a large number of ID e.g. over 1000
```

Comment:
Maybe something like this?
```suggestion
     * Putting IDs directly into a where clause instead of using a predicate was measured to be significantly
     * faster when querying a large number of ID e.g. over 1000
```
We're not really "disabling" anything, so I think this reads a lot clearer.

Reply:
Updated